nrcatalogtools/lvc.py

```diff
@@ -224,3 +224,462 @@ def get_strain_from_lvcnr_file(path_to_file,
         spin_args=spin_args,
         fixed_args=fixed_args,
     )
+
+
+
+
+##################################
+# Generate time domain waveforms
+##################################
+
+def Norm(vec_x, vec_y, vec_z):
```

Comment:
We can replace this with `https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html`, i.e.

`Norm(x, y, z)` is identical to `numpy.linalg.norm( [x, y, z] )`